test/metabase/api/activity_test.clj

```diff
@@ -148,8 +143,9 @@
     (t2/insert! QueryExecution (:card views))))
 
 (deftest popular-items-test
-  ;; Clear out the view log so that test doesn't read stale state
+  ;; Clear out the view log & query execution log so that test doesn't read stale state
   (t2/delete! :model/ViewLog)
+  (t2/delete! :model/QueryExecution)
```

Comment:
I don't remember, but I feel like I might've written this originally? 
I wonder if this same behaviour can be achieved with `(mt/with-model-cleanup [:model/ViewLog :model/QueryExecution] ...)`

Reply:
I think `with-model-cleanup` will only remove new rows that were written during the test? But these tests rely on the tables being clean at the start. I think it's OK to wipe the tables since they don't impact much else in the app.